### PR TITLE
Set isSelected property of choice input in html renderer

### PR DESCRIPTION
**Pull request by dsanvita**
https://github.com/Microsoft/AdaptiveCards/issues/346

### CODE REVIEW

**Comment by dclaux** [comment]
This is by design. To pre-select a choice, you need to set the value of the input in the JSON payload to one of the Choice input's values. There is no isSelected property on choice.

File: source/html/renderer/src/card-elements.ts [hunk 1]
```diff
@@ -839,6 +839,7 @@ export class ToggleInput extends Input {
 export class Choice {
     title: string;
     value: string;
+    isSelected: boolean;
```
Comment (dclaux) [inline source/html/renderer/src/card-elements.ts]: This is not the design. To pre-select a choice, set the Choice input's value property to one of it's choices value. This is the value pre-selection/default value is done for all input types.

Comment (dsanvita) [inline source/html/renderer/src/card-elements.ts]: I found the isSelected property in the Docs: http://adaptivecards.io/explorer/#InputChoiceSet
The docs do not mention ChoiceSet as having a value.

Do the docs need to be updated?

File: source/html/renderer/src/card-elements.ts [hunk 2]
```diff
@@ -470,7 +470,7 @@ export class FactSet extends CardElement {
             return this.speak + '\n';
```
Comment (dclaux) [inline source/html/renderer/src/card-elements.ts]: Yes, I think there is a mistake in the doc. I'll discuss with the team, thanks for reporting.

Comment (dsanvita) [inline source/html/renderer/src/card-elements.ts]: Also, I don't think the default value is implemented yet in the HTML renderer for Input.ChoiceSet

Comment (dclaux) [inline source/html/renderer/src/card-elements.ts]: You are correct. Let me double-check that isSelected is indeed not the design as the implementation will depend on that. I'll get back to you.

Comment (dclaux) [inline source/html/renderer/src/card-elements.ts]: Still checking with the team, but in the meantime I made the change to honor the default value property. Again thanks for noticing!

**Pull request closed without merging**